src/components/display/fewlines/AddIdentityForm/AddIdentityForm.tsx

```diff
@@ -10,7 +10,7 @@ import {
   ReceivedIdentityTypes,
 } from "@src/@types/Identity";
 
-export const AddIdentityInputForm: React.FC<{
+export const AddIdentityForm: React.FC<{
   type: ReceivedIdentityTypes;
   addIdentity: (identity: InMemoryTemporaryIdentity) => Promise<void>;
```

Comment:
Soooo I dunno what to chose for this one.
Should we extract that in its own type? We are kind of doing both of it in the code base.
What do you think?

Reply:
OK fixed fb44216